apps/Reaper (Demo)/install-32

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+install_packages xz-utils tar || exit 1
+cd /tmp
+wget -O reaper.tar.xz "https://www.reaper.fm/files/6.x/reaper656_linux_armv7l.tar.xz" || error "Failed to download reaper!"
+status "Extracting reaper..."
+tar -xf reaper.tar.xz || error "Failed to extract reaper!"
+cd reaper_linux_armv7l || error "Failed to cd into 'reaper_linux_armv7l'"
+status_green "Done!"
+status "Installing reaper..."
+chmod +x install-reaper.sh || error "Failed to make 'install-reaper.sh' executable!"
+sudo ./install-reaper.sh --install /opt || error "Failed to run 'sudo install-reaper.sh --install /opt'"
+sudo rm -f /usr/local/bin/reaper &>/dev/null
+sudo ln -s /opt/REAPER/reaper /usr/local/bin/reaper || error "Failed to link /opt/REAPER/reaper to /usr/local/bin/reaper!"
+mkdir -p "$HOME/.local/share/icons/hicolor/64x64/apps/" || error "Failed to make icon directory!"
+cp "${DIRECTORY}/pi-apps/apps/Reaper (Demo)/icon-64.png" "$HOME/.local/share/icons/hicolor/64x64/apps/reaper.png" || error "Failed to copy icon!"
+echo "[Desktop Entry]
+Name: Reaper
+Comment: Reaper is a complete Digital Audio Workstation, for computers, offering a full multitrack audio and MIDI recording, editing, processing, mixing and mastering toolset. 
+Exec: reaper
+Icon: $HOME/.local/share/icons/hicolor/64x64/apps/reaper.png
+Terminal: false
+Category: AudioVideo;Audio;
+Type: Application
```

Comment:
Are you sure that placing colon characters here is supported by all desktop environments? I'm used to "`Name=`", "`Exec=`", etc.

Reply:
In the `Category`? I'm pretty sure it's normal, lots of desktop files use semicolons to separate categories.

Reply:
Oh sorry, I see what you're saying. I think you're right, I'll fix it.